docs/content/what-is-a-local-activity.md

```diff
@@ -6,13 +6,19 @@ description: todo
 
 A Local Activity is an [Activity Execution](/docs/content/what-is-an-activity-execution) that executes in the same process as the [Workflow Execution](/docs/content/what-is-a-workflow-execution) that spawns it.
 
-Although a Local Activity consumes less resources than a regular Activity Execution, it is subject to shorter durations and a lack of rate limiting.
+Some Activities are very short lived and do not need the queuing semantic, flow control, rate limiting and routing capabilities.
+For this case, Temporal supports a _local Activity_ feature.
 
-Some Activities are very short lived and do not need the queuing semantic, flow control, rate limiting and routing capabilities. For this case, Temporal supports a _local Activity_ feature. Local Activities are executed in the same worker process as the Workflow that invoked them. Consider using local Activities for functions that are:
+The main benefit of local Activities is that they are much more efficient in utilizing Temporal service resources and have much lower latency overhead compared to the usual Activity invocation.
```

Comment:
```suggestion
The main benefit of local Activities is that they are much more efficient in utilizing Temporal service resources and have much lower latency overhead compared to the usual Activity Executions.
```

What does it mean to be "more efficient in utilizing Temporal service resources"? I actually don't know what that means here - any way we can be explicit?

Same thing with "latency overhead" - are we specifically referring to the fact that the Workflow Execution does not have to wait like a normal Activity Execution?

Reply:
honestly i dont love that text either, i just copy pasted it from what we had before